be/src/olap/tablet.cpp

```diff
@@ -73,8 +56,8 @@ TabletSharedPtr Tablet::create_from_tablet_meta_file(
 
     // add new fields
     boost::filesystem::path file_path_path(file_path);
-    std::string shard_path = file_path_path.parent_path().parent_path().parent_path().string();
-    std::string shard_str = shard_path.substr(shard_path.find_last_of('/') + 1);
+    string shard_path = file_path_path.parent_path().parent_path().parent_path().string();
+    string shard_str = shard_path.substr(shard_path.find_last_of('/') + 1);
```

Comment:
you delete using std::string, can this be compiled?

Reply:
using std::string in tablet.h